src/modules/hooks/useBlockExplorer/useBlockExplorer.ts

```diff
@@ -1,7 +1,11 @@
 import { useCallback } from 'react';
 import { useChains, type Config } from 'wagmi';
 
-type ChainEntityType = 'address' | 'tx' | 'token';
+export enum ChainEntityType {
```

Comment:
I'm curious, how enum would provide more secure type enforcement? I never thought about that before 🤔

Reply:
I believe it doesn't make a big difference here, how I see it is that here the enum provides a sort of abstraction of the underlying value. If the block explorer path for transactions changes from `tx` to `transaction` (which probably never happen in any case), we only need to change the enum value